### PR TITLE
Feature/2

### PR DESCRIPTION
**Pull request by Lana2683**
Implemented hometask-3:
Evaluation criteria:
* Markup is done with React Components and React Fragments (parent-child); 
* Apply styles (no need to do pixel perfect and strict colors following); 
* Use PropTypes; 
* Use <ErrorBoundary> component for catching and displaying errors.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import Homepage from './Components/Homepage.js';
+
+export const App = () =>
+    <>
```
Comment (mmarkelov) [inline src/App.js]: I'm not sure if you need this fragment here

File: src/Components/ErrorBoundary.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+export const ErrorBoundary = ({ children }) => {
```
Comment (mmarkelov) [inline src/Components/ErrorBoundary.js]: You should use [`<ErrorBoundary/>`](https://reactjs.org/docs/error-boundaries.html) component to catch possible errors

File: src/App.js (hunk 1 above)
Comment (Lana2683) [reply src/App.js]: fixed

File: src/Components/ErrorBoundary.js (hunk 2 above)
Comment (Lana2683) [reply src/Components/ErrorBoundary.js]: fixed

File: src/Components/ErrorBoundary.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+class ErrorBoundary extends Component {
```
Comment (mmarkelov) [inline src/Components/ErrorBoundary.js:4]: You also need to define `getDerivedStateFromError` method

Comment (Lana2683) [reply src/Components/ErrorBoundary.js:4]: fixed

**Pull request merged**